### PR TITLE
LPD-28472 Add Test Automation for LPP-52725.

### PR DESCRIPTION
**Pull request by manuelRives**
https://liferay.atlassian.net/browse/LPD-28472

Adds tests

### CODE REVIEW

**Comment by manuelRives** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:playwright

File: modules/test/playwright/tests/document-library-web/dmInstanceSettings.spec.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {documentLibraryPagesTest} from './fixtures/documentLibraryPagesTest';
+
+export const CheckOverwriteMaxSizetest = mergeTests(
+	documentLibraryPagesTest,
+	isolatedSiteTest,
+	loginTest()
+);
```
Comment (adolfopa) [inline modules/test/playwright/tests/document-library-web/dmInstanceSettings.spec.ts:16]: All usages of `mergeTests` I'e seen assign the result to a variable with regular capitalization (e.g. `checkOverwriteMaxSizeTest`).  In any case, why not keep it simple and name it `test`?

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/test/playwright/tests/document-library-web/dmInstanceSettings.spec.ts (hunk 1 above)
Comment (boton) [reply modules/test/playwright/tests/document-library-web/dmInstanceSettings.spec.ts:16]: +1 to keep simply `test` any case `camelCase` as @adolfopa says

Comment (manuelRives) [reply modules/test/playwright/tests/document-library-web/dmInstanceSettings.spec.ts:16]: Perfect, no problem, I'll change it to test. Thanks for the advice.

Comment (manuelRives) [reply modules/test/playwright/tests/document-library-web/dmInstanceSettings.spec.ts]: Changed

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:test:relevant

**Review by boton — APPROVED** [review]
LGTM

It's a little flaky but it's unrelated to this test 😄 

https://testray.liferay.com/#/project/34754/routines/34756/build/54323475/case-result/54324260

image

**Comment by boton** [comment]
ci:forward

**Comment by boton** [comment]
ci:stop

File: modules/test/playwright/tests/document-library-web/dmInstanceSettings.spec.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {documentLibraryPagesTest} from './fixtures/documentLibraryPagesTest';
+
+export const test = mergeTests(
```
Comment (boton) [inline modules/test/playwright/tests/document-library-web/dmInstanceSettings.spec.ts:12]: This `export` isn't needed 😕 we are not importing it anywhere

**Review by boton — CHANGES_REQUESTED** [review]
All it's almost done and LGTM but I added a little inline comment to change, sorry. 

When it is done feel free to forward.

File: modules/test/playwright/tests/document-library-web/dmInstanceSettings.spec.ts (hunk 2 above)
Comment (manuelRives) [reply modules/test/playwright/tests/document-library-web/dmInstanceSettings.spec.ts:12]: I am checking that export and it looks like it is used by the test itself.

In fact, as far as I can see all the other tests have their associated export test. So it looks like it is being used.

Comment (manuelRives) [reply modules/test/playwright/tests/document-library-web/dmInstanceSettings.spec.ts]: Now I see, I thought you were referring to the entire mergeTests block. I have already removed the export.

Thanks for the advice.

Fixed.

**Comment by manuelRives** [comment]
ci:forward

**Pull request closed without merging**